### DISCUSSION TITLE
Release 0.3.1 (1.19.80.21)

### DISCUSSION THREAD

**Discussion by minecraftchloe** [Mojang/minecraft-editor#86] [Announcements]
_[Read the previous release notes here](https://github.com/Mojang/minecraft-editor/discussions/2)_

---

## Summary
Wow!! Just...wow!!!! Thank you for an incredible first week in Preview! 🥹 

Your excitement, helpful feedback, and how you've already been able to start using the API ... despite it not having any official documentation nor being ready yet ... has blown our minds! 🤯 

_Disclaimer:_ As our small team is heads-down working towards v0.4: The API Update, which will include documentation and a starter-kit, any bug and quality fixes week to week will be very modest in size and scope. 

---

## Fixed this week
- Custom blocks are now listed in the block selector: Previously if you had a pack with custom blocks, they wouldn't show up in the block dropdown. Now they do! 
- [Fixed z-fighting](https://github.com/Mojang/minecraft-editor/discussions/19) on paste preview over selection volume.

---

## Next
v0.4: The API Update will be focused on exposing the Editor APIs for you to begin building your own Editor extensions. For now we invite you to acquaint yourself with the existing [scripting](https://docs.microsoft.com/en-us/minecraft/creator/scriptapi/) efforts and get familiar with Typescript!

Additionally, if you're new to the Editor GitHub this week, we encourage you to join the Bedrock Add-Ons Discord [#Editor channel](https://discord.gg/nkGxN62z)!

---

Thank you for being here! You're the best! ❤️

**Comment by ihategravel2** [comment]
Custom Blocks showed in the drop-down menu in the last preview, you just had to put the prefix first.
